src/App.css

```diff
@@ -1,38 +1,58 @@
-.App {
-  text-align: center;
+.Nav-bar {
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  padding: 0.5em 0 0.5em 4rem;
+  border: 1px solid #a2a2a2;
+  background-color: #f4f4f4;
+  -webkit-box-shadow: 0 0 14px 0 rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 0 0 14px 0 rgba(0, 0, 0, 0.75);
+  box-shadow: 0 0 14px 0 rgba(0, 0, 0, 0.75);
+  -webkit-border-radius: 5px;
+  -moz-border-radius: 5px;
+  border-radius: 5px;
+  justify-content: space-between;
+}
+
+.icon-continer {
+  display: flex;
+  gap: 5px;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.logo-image {
+  width: 3rem;
+  height: 3rem;
+}
+
+.options-continer {
+  display: flex;
+  list-style: none;
+  gap: 20px;
+  margin: 24px 20px 0 0;
```

Comment:
Following the RWD guidelines :page_with_curl: , please avoid using px :x:  and instead measurement units like `rem, em and percentages (%)` :heavy_check_mark:

Reply:
you are totally right i will change it